packages/twenty-server/src/workspace/workspace-query-runner/jobs/call-webhook-jobs.job.ts

```diff
@@ -54,27 +49,45 @@ export class CallWebhookJobsJob
       await this.workspaceDataSourceService.connectToWorkspaceDataSource(
         data.workspaceId,
       );
-    const operationName = `${data.operation}.${objectMetadataItem.namePlural}`;
+    const nameSingular = data.objectMetadataItem.nameSingular;
+    const operation = data.operation;
+    const eventType = `${operation}.${nameSingular}`;
     const webhooks: { id: string; targetUrl: string }[] =
       await workspaceDataSource?.query(
-        `SELECT * FROM ${dataSourceMetadata.schema}."webhook" WHERE operation='${operationName}'`,
+        `
+            SELECT * FROM ${dataSourceMetadata.schema}."webhook" 
```

Comment:
So from your code my understanding is we can store operations such as
`create.company` => create API events on Company object
`*.company` => All API events. on Company object
`create.*` => All create API events regardless of the object
`*.*` => All events.
Why do we need a LIKE %% in this SQL query?

Reply:
because we want to be able to add multiple triggers like `create.company,update.*`